src/cancellationToken/cancellationToken.ts

```diff
@@ -60,7 +60,7 @@ function createCancellationToken(args: string[]): ServerCancellationToken {
     }
     else {
         return {
-            isCancellationRequested: () => pipeExists(cancellationPipeName!), // TODO: GH#18217
+            isCancellationRequested: () => pipeExists(cancellationPipeName), // TODO: GH#18217

```

Comment:
```suggestion
            isCancellationRequested: () => pipeExists(cancellationPipeName),

```

Reply:
Ah, yeah, indeed; I did just this in #57099 which I pulled out to show specifically what is new in the latest TS.